litho-core/src/main/java/com/facebook/litho/LogTreePopulator.java

```diff
@@ -50,7 +66,7 @@ public static PerfEvent populatePerfEventFromLogger(
 
     perfEvent.markerAnnotate(FrameworkLogEvents.PARAM_LOG_TAG, logTag);
 
-    @Nullable final TreeProps treeProps = c.getTreeProps();
+    @Nullable final TreeProps treeProps = c.getTreeProps() == null ? savedTreeProps : c.getTreeProps();
```

Comment:
My previous comment in the original PR was about making logic of passing `treeProps` similar to other lib components, like `ComponentContext`. Pass nullable `treeProps` as a parameter to this method, if they are `null`, fallback to `c.getTreeProps()`.

Reply:
Thanks, fixed.